Bio/Align/__init__.py

```diff
@@ -308,7 +311,20 @@ def __repr__(self):
         # return "%s(%s, %s)" \
         #       % (self.__class__, repr(self._records), repr(self._alphabet))
 
-    def format(self, format):
+    def format(self, format_spec):
+        """Return the alignment as a string in the specified file format [DEPRECATED].
+
+        This method is deprecated; instead of alignment.format(format_spec),
+        please use format(alignment, format_spec).
+        """
+        warnings.warn("""\
```

Comment:
You don't need to use triple quotes with embedded new lines, just a long multiline string with implicit concatenation is enough:

```python
warnings.warn(
    "alignment.format has been deprecated, and we intend to remove it in a future "
    "release of Biopython. Instead of alignment.format(format_spec), please use "
    "format(alignment, format_spec).",
    BiopythonDeprecationWarning
)

```

Reply:
If you want to leave this for now, we can probably do a sweep of the entire code base for this next?